pykov.py

```diff
@@ -1085,7 +1085,13 @@ def walk(self, steps, start=None, stop=None):
         ['B', 'A', 'A', 'A', 'A', 'A', 'B']
         """
         if start is None:
-            start = self.steady().choose()
+            steady = self.steady()
```

Comment:
You mentioned:
> otherwise the starting state is unpredictable.

Do you want to change the documentation for `.steady()`, to indicate that the returned array is unpredictable, or `.walk()`, to indicate that a state is chosen randomly?

Either way, I think the warning is a good idea. Documenting the non-ergodic case a little more would be nice since we can't detect it and issue a warning directly.